languages.toml

```diff
@@ -1,5 +1,6 @@
 [[language]]
 name = "rust"
+display-name = "Rust"
```

Comment:
Can we just use `name`? It won't look as good, but at least it'll match the name used inside the editor.

Other than that LGTM 👍🏻

Reply:
I was thinking we could also use it for #976, but I'm not particularly attached to it.

Reply:
Removed `display-name` in favor of `name` anyways :P